### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by nvictus**
## Description

What was changed in this pull request?

Dev
* Pin dependencies on `jupyterlab` and `ipywidgets`. Currently, custom widgets fail in jlab 1.0 and also break if ipywidgets isn't just right.
* Update makefile

Client
* Deep copy options dicts passed to `Track()`
* Expose `viewconf.views` as a list using a property descriptor

Why is it necessary?

Make `Track` objects more robust and `View`s more easily accessible from `ViewConf`s.

## Checklist

- [ ] Unit tests added or updated
- [ ] Documentation added or updated
- [ ] Updated CHANGELOG.md

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,44 +1,58 @@
...
+
 build:
 	python setup.py jsdeps
 
-clean:
+clean-py:
+	rm -rf build/
+
+clean-js:
 	rm -rf higlass/static/
```
Comment (flekschas) [inline Makefile]: this seems to be unrelated to javascript or am I missing something? it probably doesn't hurt though

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
+-r requirements.txt
 numpy
 ipython
 jupyter
-jupyterlab
+jupyterlab<1.0
```
Comment (flekschas) [inline requirements-dev.txt]: 😬

File: requirements.txt [hunk 3]
```diff
@@ -3,7 +3,7 @@ cytoolz
 flask
 flask-cors
 fusepy
-ipywidgets
+ipywidgets==7.4.2
```
Comment (flekschas) [inline requirements.txt]: 😬

File: Makefile (hunk 1 above)
Comment (nvictus) [reply Makefile]: `higlass/static/` is where the final extension js is dumped that ships with `higlass-python`; `build/` is where everything goes when publishing a source tarball of the python package
